onnx/inliner/inliner.cc

```diff
@@ -492,6 +480,9 @@ void InlineFunctions(
             callee.name().c_str());
         ConvertVersion(*model, node, callee, target_version);
       }
+      // Append valueinfos of called function
+      for (auto& callee_vi : callee.value_info())
+        *value_infos.Add() = callee_vi;
```

Comment:
Follow up on https://github.com/onnx/onnx/pull/5942/files#r1498435308, will this append value_info of function inputs outputs?

Reply:
Good point. Maybe we should not insert duplicates (which can happen only for inputs/outputs).

Reply:
Done